src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -22,42 +19,38 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         ''' </summary>
         Private ReadOnly _tupleElementIndex As Integer
 
-        Private ReadOnly _locations As ImmutableArray(Of Location)
         Private ReadOnly _useSiteDiagnosticInfo As DiagnosticInfo
-        Private ReadOnly _correspondingDefaultField As TupleErrorFieldSymbol
 
         ' default tuple elements like Item1 Or Item20 could be provided by the user or
         ' otherwise implicitly declared by compiler
-        Private ReadOnly _isImplicitlyDeclared As Boolean
-
-        Public Sub New(container As NamedTypeSymbol,
-                       name As String,
-                       tupleElementIndex As Integer,
-                       location As Location,
-                       type As TypeSymbol,
-                       useSiteDiagnosticInfo As DiagnosticInfo,
-                       isImplicitlyDeclared As Boolean,
-                       correspondingDefaultFieldOpt As TupleErrorFieldSymbol)
+        'Private ReadOnly _isImplicitlyDeclared As Boolean
+
+        Public Sub New(
```

Comment:
@jcouv 
It a style I'm being to adapt to as it really easy read (specially for VB).
Align the opening and closing braces, then align the parameter names 1 space after the opening brace. One parameter per a line. As well being convenient to add comments on the paremeters (if need).
eg
```vbnet
        Public Sub New(
                        container As NamedTypeSymbol,
                        name As String, ' Non Null
                        tupleElementIndex As Integer,
                        location As Location,
                        type As TypeSymbol,
                        useSiteDiagnosticInfo As DiagnosticInfo,
                        isImplicitlyDeclared As Boolean,
                        correspondingDefaultFieldOpt As TupleErrorFieldSymbol
                      )

```